pybossa/view/projects.py

```diff
@@ -3188,9 +3189,21 @@ def coowners(short_name):
         if form.user.data:
             # search users
             query = form.user.data
+            params = json.loads(request.data)
 
             filters = {'enabled': True}
+
+            # Search all users.
             users = user_repo.search_by_name(query, **filters)
+
+            # If searching contacts, filter results to only coowners.
+            if params.get('contact'):
+                filtered_list = []
+                for user in users:
+                    if user.id in project.owners_ids:
+                        filtered_list.append(user)
+                users = filtered_list
```

Comment:
If you prefer, line 3201 to 3205 can be rewrite with list comprehension.
```python
users = [user for user in users if user.id in project.owners_ids]
```

Reply:
Fixed via https://github.com/bloomberg/pybossa/pull/776/commits/c8440d739b742e8c85e9f861a17aa0c4c272d34d